pkg/crd/kubermatic/v1/cluster.go

```diff
@@ -455,11 +455,12 @@ type MLASettings struct {
 }
 
 type ComponentSettings struct {
-	Apiserver         APIServerSettings       `json:"apiserver"`
-	ControllerManager ControllerSettings      `json:"controllerManager"`
-	Scheduler         ControllerSettings      `json:"scheduler"`
-	Etcd              EtcdStatefulSetSettings `json:"etcd"`
-	Prometheus        StatefulSetSettings     `json:"prometheus"`
+	Apiserver          APIServerSettings       `json:"apiserver"`
+	ControllerManager  ControllerSettings      `json:"controllerManager"`
+	Scheduler          ControllerSettings      `json:"scheduler"`
+	Etcd               EtcdStatefulSetSettings `json:"etcd"`
+	Prometheus         StatefulSetSettings     `json:"prometheus"`
+	NodePortProxyEnvoy NodeportProxyComponent  `json:"nodePortProxyEnvoy"`
```

Comment:
By popular opinion, since we already merged CRD migration code, no new changes in `pkg/crd/kubermatic/v1` is expected.

Reply:
Where do you suggest this should go?

Reply:
Only in pkg/apis/.

Reply:
Done